src/scss/bootstrap-view.scss

```diff
@@ -0,0 +1,94 @@
+.btn-primary, .dropdown-menu {
+  min-width: 140px;
+  border-color: $gray-900;
+  font-size: 13px;
+  -webkit-border-radius: 1px;
+  -moz-border-radius: 1px;
+  border-radius: 1px;
+}
+
+.dropdown-menu {
+  padding: 0;
+  margin: 0.5rem 0 0;
+
+  .dropdown-item {
+    padding: 0.6rem 0.7rem;
+
+    &:hover, &:focus, &.active {
+      outline: none !important;
+      box-shadow: none !important;
+      background-color: $white !important;
+      cursor: pointer;
+      color: $gray-900;
+    }
+  }
+
+  button+button {
+    border-top: 1px solid #333333;
+  }
+}
+
+.dropdown-toggle {
+
+  &:after{
+    margin-left: 2.4rem;
+  }
+}
+.show > .btn-primary.dropdown-toggle, .btn-primary:hover {
+  background-color: #ffffff !important;
+  border-color: $gray-900 !important;
+  outline: none !important;
+  box-shadow: none !important;
+}
+
+.head-nav {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+  z-index: 10;
+}
+
+h2{
+  background-color: $gray-200;
+  font-size: 12px;
+}
+
+.overlay {
+  border: none !important;
+}
+
+.timeline {
```

Comment:
This should go to map.component.scss, where the .timeline element is actually used. This applies to most of this file.
Please place the rules closer to the elements they are styling.